test/linearAlgebra/CSRMatrix.cpp

```diff
@@ -74,20 +74,20 @@ TEST_CASE("CSRMatrix")
         // Dense
         NeoFOAM::Field<NeoFOAM::scalar> checkDense(exec, 9);
         auto checkDenseSpan = checkDense.span();
-        auto csrDenseSpan = denseMatrixConst.span();
+        auto csrDenseView = denseMatrixConst.view();
```

Comment:
I guess it should be just `denseView`?
```suggestion
        auto denseView = denseMatrixConst.view();
```